esp/services/esdl_svc_engine/esdl_binding.hpp

```diff
@@ -263,6 +263,11 @@ class EsdlBindingImpl : public CHttpSoapBinding
     void getRequestContent(IEspContext &context, StringBuffer & req, CHttpRequest* request, const char * servicename, const char * methodname, const char *ns, unsigned flags);
     void setXslProcessor(IInterface *xslp){}
 
+    int getMethodProperty(IEspContext &context, const char *serv, const char *method, StringBuffer &page, const char *propname, const char *dfault);
+
+    virtual int getMethodDescription(IEspContext &context, const char *serv, const char *method, StringBuffer &page) override;
+    virtual int getMethodHelp(IEspContext &context, const char *serv, const char *method, StringBuffer &page) override;
```

Comment:
The commit message states the description and help "can be looked up as needed. Does the absence of any calls to these new methods mean they aren't needed, or should they be called from somewhere?

Reply:
These are virtual overrides that are already called by ESP when needed to display these bits of text on forms etc.

Reply:
Yes.  I refactored my fix without updating the log text.  Good catch.